tests/lang-python.yaml

```diff
@@ -47,7 +47,7 @@
   command:
     [
       ver=3.11.4; origdir="$PYENV_ROOT/versions/$ver" newVdir="/workspace/.pyenv_mirror/fakeroot/versions";
-      pyenv install $ver && test -e "$origdir" && mkdir -p "$newVdir" && mv "$origdir" "$newVdir" && GITPOD_REPO_ROOT=/workspace bash -ci "pyenv install -s $ver && pip install cowsay && pyenv global $(pyenv version-name) $ver && cowsay Gitpod",
+      pyenv install $ver && test -e "$origdir" && mkdir -p "$newVdir" && mv "$origdir" "$newVdir" && GITPOD_REPO_ROOT=/workspace bash -ci "pyenv install -s $ver && pip install cowsay && pyenv global $(pyenv version-name) $ver && cowsay --text Gitpod",
```

Comment:
Good catch, @filiptronicek , I added something like this earlier, but dropped it when rebasing. Good luck on the run! :crossed_fingers: Please rerequest my review or ping me if/when needed.

Reply:
It looks like it's timing out again 🙄

Reply:
testing with the latest of Python 2 now, let's just see how this goes...

Reply:
It works with Python 2. We shouldn't use it though, so I will merge the change without updating to 3, but will investigate it again to see what's up with the timeouts for the sake of not halting the base image updates.

Reply:
Yeah, that's a good compromise.

@axonasif do you recall why the test was written with Python2?

Reply:
I think it was mostly about checking whether it breaks on something so old.